### PR TITLE
mo(react-native): Debug two iOS apps simultaneously

### PR DESCRIPTION
**Pull request by xavierlefevre**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/mo/debug-two-ios-apps-simultaneously/react-native/debugging/debug-two-ios-apps-simultaneously.mo.md">debug-two-ios-apps-simultaneously.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by xavierlefevre** [comment]
@tychota : il aime pas mon `* 👨‍💻 **Owner:** ` Danger

**Comment by louiszawadzki** [comment]
@tychota Shall we introduce the emoji in the owner?

@xavierlefevre  I personnally prefer consistency across all the standards and not having something a little bit different everytime - that's even the point of standards ;)

 @xavierlefevre Can we either change the danger file or change `* 👨‍💻 **Owner:**` into `## Owner:`? And I'm not sure we need the last update as we have the git history for this purpose.

**Comment by xavierlefevre** [comment]
@louiszawadzki : I am not going to argue about the emoji, removed it. Just personally liked it more :-D
About the date time, Git has the history, people reading the generated Gitbook don't ! It's important to know how old is a standard as things change fast. @tychota what about adding it to Danger ?

@louiszawadzki : Can you validate this PR that we merge it? And bring some value? :D

File: react-native/debugging/debug-two-ios-apps-simultaneously.mo.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# [MO] Debug two iOS apps simultaneously
+
+## Owner: [Xavier Lefèvre](https://www.github.com/xavierlefevre)
+
+## Last update date: 29th of January 2018
```
Comment (tychota) [inline react-native/debugging/debug-two-ios-apps-simultaneously.mo.md:5]: OK pour moi, mais comment s'assurer que ça soit le cas partout et que ça reste à jour.

On a fait ça sur les  std déprécié, c'est ptet plus maintenable que la date : https://bamtech.gitbook.io/dev-standards/react-native/setup/mo-setup-and-deploy-new-project-to-staging-with-hockeyapp-deprecated

File: react-native/debugging/debug-two-ios-apps-simultaneously.mo.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* Using [React Native Debugger](https://github.com/jhen0409/react-native-debugger)
+
+### Steps
+
+* For the first application, you can launch it as you would normally:
+
+  * Open it with xcode `xcode ./ios/your-first-app.xcworkspace`
+  * Open React Native Debugger
+  * Launch the simulator from xcode with the play button on the top left
+
```
Comment (tychota) [inline react-native/debugging/debug-two-ios-apps-simultaneously.mo.md:19]: Add a check:

```
{% hint style='success' %} **CHECK**

In RN debugger, you can see log from first app.

{% endhint %}
```

Ca va produire un truc joli sur gitbook genre comme sur ce standard: https://bamtech.gitbook.io/dev-standards/ops/mo-setup-https-on-your-docker-environment

File: react-native/debugging/debug-two-ios-apps-simultaneously.mo.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    * Look for "8081"
+    * Replace "8081" by a new port, like "9980"
+    * Save each change
+  * In React Native Debugger:
+    * Open a new window "Cmd + T"
+    * Enter your new port "9980"
+  * Launch the simulator from xcode with the play button
+  * Launch a new packager from your project directory: `react-native start --port 9980`
+  * Close and re-open the app from within the simulator
+
```
Comment (tychota) [inline react-native/debugging/debug-two-ios-apps-simultaneously.mo.md:38]: Add a check:

```
{% hint style='success' %} **CHECK**

In RN debugger, you can see log from second app.

{% endhint %}
```

File: react-native/debugging/debug-two-ios-apps-simultaneously.mo.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  * Open React Native Debugger
+  * Launch the simulator from xcode with the play button on the top left
+
+* For the second application:
+
+  * Open it with xcode `xcode ./ios/your-second-app.xcworkspace`
+  * Modify the port used by the packager in the native code:
+    * Do a full project search with "Maj + Cmd + F"
+    * Look for "8081"
+    * Replace "8081" by a new port, like "9980"
```
Comment (tychota) [inline react-native/debugging/debug-two-ios-apps-simultaneously.mo.md:30]: There is no way to configure it without hardcoding ?

**Review by tychota — APPROVED** [review]

File: react-native/debugging/debug-two-ios-apps-simultaneously.mo.md (hunk 4 above)
Comment (xavierlefevre) [reply react-native/debugging/debug-two-ios-apps-simultaneously.mo.md:30]: No clue

File: react-native/debugging/debug-two-ios-apps-simultaneously.mo.md (hunk 3 above)
Comment (xavierlefevre) [reply react-native/debugging/debug-two-ios-apps-simultaneously.mo.md:38]: Done !! :-)

File: react-native/debugging/debug-two-ios-apps-simultaneously.mo.md (hunk 2 above)
Comment (xavierlefevre) [reply react-native/debugging/debug-two-ios-apps-simultaneously.mo.md:19]: Done !! :-)

**Pull request merged**